projects/collection-editor-library/src/lib/components/question/question.component.ts

```diff
@@ -555,6 +555,17 @@ export class QuestionComponent implements OnInit, AfterViewInit, OnDestroy {
 
     // to handle when question type is mcq
     if (this.questionInteractionType === 'choice') {
+      const data = _.get(this.treeService.getFirstChild(), 'data.metadata');
```

Comment:
@pradeepkumarcm96 `this.treeService.getFirstChild()` function you calling multiple times? Can we calls `ngOnit` once and use other places?